filament/backend/src/opengl/platforms/PlatformEGL.cpp

```diff
@@ -113,6 +129,18 @@ Driver* PlatformEGL::createDriver(void* sharedContext) noexcept {
             EGL_RECORDABLE_ANDROID, 1,                          // 12
             EGL_NONE                                            // 14
     };
+#else
+   EGLint configAttribs[] = {
+          EGL_SURFACE_TYPE, EGL_PBUFFER_BIT,
+          EGL_RED_SIZE, 8,
+          EGL_GREEN_SIZE, 8,
+          EGL_BLUE_SIZE, 8,
+          EGL_ALPHA_SIZE,  0,
+          EGL_DEPTH_SIZE, 8,
```

Comment:
8 bits depth? I don't think that's right at all. This should be at least 16, preferably 24 or 32.